src/controllers/expenses.js

```diff
@@ -0,0 +1,93 @@
+'use strict';
+
+const { expensesService } = require('../services/expenses');
+const { usersService } = require('../services/users');
+
+const getAll = (req, res, next) => {
+  const expenses = expensesService.getAll();
+
+  if (!Object.keys(req.query)) {
+    res.send(expenses);
+
+    return;
+  }
+
+  next();
+};
+
+const getAllFilteredExpenses = (req, res) => {
+  const filters = req.query;
+
+  const expensesToShow = expensesService
+    .getAllFilteredExpenses(filters);
+
+  res.send(expensesToShow);
+};
+
+const getById = (req, res) => {
+  const { expenseId } = req.params;
+  const foundExpense = expensesService.getById(expenseId);
+
+  if (!foundExpense) {
+    res.sendStatus(404);
+
+    return;
+  }
+
```

Comment:
I think here we have also check for incorrect expenseId type (isNaN(+expenseId)) and send status 400